src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -392,6 +396,11 @@ public void setPerTargetOutput(final File perTargetOutput) {
             this.perTargetOutput = perTargetOutput;
         }
 
+        /** If set, the metrics collector will output per target coverage information to this file. */
```

Comment:
I realize the phrasing of this is because the other one is the same but why are these "if set" and the other setter is "sets"? Just seems weird :)